step-capstone/src/components/TravelObjectForms/AddEvent.js

```diff
@@ -61,9 +61,10 @@ export default function AddEvent(props) {
             startDate: startDate,
             endDate: endDate,
             finalized: checked,
-            location: location.address,
-            coordinates: location.coordinates,
-            description: description
+            location: location ? location.address : null,
```

Comment:
maybe rename this  variable `location` to `address` to distinguish it with  the location object which contains address?
so this line of code will be `address: location? location.address: null`

Reply:
Good point, will do!